admin/client/src/boot/index.js

```diff
@@ -37,8 +41,24 @@ function appBoot() {
 
   // Set the initial config state.
   window.store.dispatch(configActions.setConfig(window.ss.config));
+
+
+  // Enable BS Popovers and tooltips
+  $(function () {
```

Comment:
I think this should be placed in the respective ReactJS component containing the tooltip logic - not every component will be rendered at the time `boot.js` is run, just happens to be the case here.